wrap/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 3.9)
```

Comment:
So now it works even without installing, correct?

I mean, this is not installed along with GTSAM, or it is? Maybe we need more testing if it will go with every GTSAM installation...

Reply:
For GTSAM yes, the behavior is the same as the old one. If we follow the standard cmake process for GTSAM, everything should be the same.

The cool thing is that one can also install `wrap` separately now in case they have a project that doesn't need GTSAM.

Reply:
Cool!